src/gui/SubWindow.cpp

```diff
@@ -220,22 +235,34 @@ void SubWindow::resizeEvent( QResizeEvent * event )
 	{
 		buttonBarWidth = buttonBarWidth + m_buttonSize.width() + buttonGap;
 		m_maximizeBtn->move( middleButtonPos );
+	// on Mac the isMaximize() function returns always false
```

Comment:
Mentioning an unused function can confused people when they read this later.

1. Should we be using isMaximize() instead of `WindowMaximizeButtonHint`?
2. If so, should we override this on Mac only and leave the logic identical between function calls?

The reason I ask is that the logic is for Mac-only yet we allow some portions to execute every resize event.  Is that desired?